scene/animation/tween.cpp

```diff
@@ -896,7 +896,12 @@ real_t Tween::get_runtime() const {
 			runtime = t;
 	}
 	pending_update--;
-	return runtime;
+
+	if (speed_scale == 0) {
```

Comment:
You could spare an unnecessary for loop by moving this to the top of the function.